src/Symfony/Component/VarDumper/Tests/Caster/ExceptionCasterTest.php

```diff
@@ -354,4 +355,33 @@ public function testAnonymous()
 
         $this->assertDumpMatchesFormat($expectedDump, $e, Caster::EXCLUDE_VERBOSE);
     }
+
+    /**
+     * @requires function \Symfony\Component\ErrorHandler\Exception\FlattenException::create
```

Comment:
`@requires` FTW

Reply:
Oh! Nice. I check for `requires class` but it didn't exist. Nice usage of "function"